cvat/apps/engine/serializers.py

```diff
@@ -2170,3 +2177,120 @@ def _update_assets(guide):
     class Meta:
         model = models.AnnotationGuide
         fields = ('id', 'task_id', 'project_id', 'markdown', )
+
+class UserIdentifiersSerializer(BasicUserSerializer):
+    class Meta(BasicUserSerializer.Meta):
+        fields = (
+            "id",
+            "username",
+        )
+
+class RQJobOperationSerializer(serializers.Serializer):
+    type = serializers.CharField(read_only=True)
+    target = serializers.ChoiceField(choices=["project", "task", "job"], read_only=True)
+    project_id = serializers.IntegerField(required=False, allow_null=True, read_only=True)
+    task_id = serializers.IntegerField(required=False, allow_null=True, read_only=True)
+    job_id = serializers.IntegerField(required=False, allow_null=True, read_only=True)
+    format = serializers.CharField(required=False, allow_null=True, read_only=True)
+    name = serializers.CharField(source="id", read_only=True)  # TODO: duplicated field
+
+    def to_representation(self, rq_job: RQJob) -> Dict[str, Any]:
+        try:
+            parsed_rq_id = RQIdManager.parse(rq_job.id)
+        except ValueError as ex:
+            raise serializers.ValidationError(f"RQ job ID does not match supported template: {rq_job.id}") from ex
+
+        return {
+            "type": ":".join(
+                [
+                    parsed_rq_id.action,
+                    parsed_rq_id.subresource or parsed_rq_id.resource,
+                ]
+            ),
+            "target": parsed_rq_id.resource,
+            "project_id": rq_job.meta["project_id"],
+            "task_id": rq_job.meta["task_id"],
+            "job_id": rq_job.meta["job_id"],
+            "format": parsed_rq_id.format,
+            "name": rq_job.id,
+        }
+
+
+class RQJobDetailsSerializer(serializers.Serializer):
+    status = serializers.SerializerMethodField()
+    message = serializers.SerializerMethodField()
+    id = serializers.CharField(read_only=True)
+    operation = RQJobOperationSerializer(source="*")
+    percent = serializers.SerializerMethodField()
+    enqueue_date = serializers.DateTimeField(source="enqueued_at", read_only=True)
+    start_date = serializers.DateTimeField(
+        required=False, allow_null=True, source="started_at", read_only=True
+    )
+    finished_date = serializers.DateTimeField(
+        required=False, allow_null=True, source="ended_at", read_only=True
+    )
+    expire_date = serializers.SerializerMethodField()
```

Comment:
```suggestion
    expiry_date = serializers.SerializerMethodField()
```
Or `expiration`, both would be more correct from the language viewpoint.